package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "newrelic",
-      "version": "8.1.0",
+      "version": "8.2.0",
```

Comment:
odd that wasn't updated with the release.

Reply:
it did, not sure if this was because I didn't rebase properly? https://github.com/newrelic/node-newrelic/blob/main/package-lock.json#L3